### PR TITLE
Various Code Consistency Changes in Schedule.

### PR DESCRIPTION
**Pull request by wito**
This feature updates the code in `Schedule.cpp` and `KeywordHandlers.cpp` to be overall more consistent, mostly in terms of whitespace and braces, as well as other minor consistency fixes (such as normalising to `std::size_t`).

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp [hunk 1]
```diff
@@ -134,7 +134,7 @@ namespace {
         this->updateNetwork(ext_network, handlerContext.currentStep);
     }
 
-    void Schedule::handleCOMPDAT(const HandlerContext& handlerContext, const ParseContext& parseContext, ErrorGuard& errors) {
+    void Schedule::handleCOMPDAT  (const HandlerContext& handlerContext, const ParseContext& parseContext, ErrorGuard& errors) {
```
Comment (blattms) [inline src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:137]: this is the only commit that i looked at ("tabulated" made me curious.)

I have never seen different amounts of whitespace between function name and the parenthesis. Why would I want that?

Comment (wito) [reply src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:137]: Try `grep 'void Schedule::handle' src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp`.

Comment (atgeirr) [reply src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:137]: I admit it looks better. However, the convention that the parenthesis immediately follows a function name is well established in the code, so I would wish that could be kept.

Comment (bska) [reply src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:137]: > I admit it looks better. However, the convention that the parenthesis immediately follows a function name is well established in the code, so I would wish that could be kept.

Besides, if we ever get started on our [ClangFormat](https://github.com/OPM/opm-grid/pull/386) experiment in earnest, all of this discussion is moot.

**Comment by wito** [comment]
jenkins build this please

**Pull request merged**